bin/yaml/libraries.yaml

```diff
@@ -43,6 +43,14 @@ libraries:
       targets:
       - '1.2'
       type: github
+    beman_optional26:
```

Comment:
you only have to add this if you use releases

Reply:
Today I don't have a release, but soon we'll have. So I remove it or keep for future changes?

Reply:
remove now, add later

Reply:
removed